tensorflow/python/keras/saving/hdf5_format.py

```diff
@@ -229,11 +206,19 @@ def convert_custom_objects(obj):
       optimizer = optimizers.deserialize(
           optimizer_config, custom_objects=custom_objects)
 
+      def deserialize_many(deserialize_fn, config):
+        if isinstance(config, list):
```

Comment:
I would recommend extending this to non-list objects, using `nest.flatten` and its inverse.